### PR TITLE
Bump kurl_proxy dependencies

### PR DESCRIPTION
**Pull request by emosbaugh**
I'm bumping a bunch of dependencies here in favor of dependabot prs

### CODE REVIEW

File: kurl_proxy/cmd/main.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func getHttpsServer(upstream, dexUpstream *url.URL, tlsSecretName string, secret
 			} else {
 				delete(secret.Annotations, "acceptAnonymousUploads")
 			}
-			_, err = secrets.Update(secret)
+			_, err = secrets.Update(context.Background(), secret, metav1.UpdateOptions{})
```
Comment (emosbaugh) [inline kurl_proxy/cmd/main.go:320]: This is after the request context is closed so it must be background

https://github.com/replicatedhq/kots/pull/1749/files#diff-db27cebad99585565557ed130999dc35ca6e4d2e4cf648b04119580339d9d407R297

File: kurl_proxy/cmd/main.go [hunk 2]
```diff
@@ -392,7 +392,7 @@ func getHttpsServer(upstream, dexUpstream *url.URL, tlsSecretName string, secret
...
```
Comment (emosbaugh) [inline kurl_proxy/cmd/main.go:395]: This is after the request context is closed so it must be background

https://github.com/replicatedhq/kots/pull/1749/files#diff-db27cebad99585565557ed130999dc35ca6e4d2e4cf648b04119580339d9d407R378

File: kurl_proxy/cmd/main.go [hunk 3]
```diff
@@ -279,14 +279,14 @@ func getHttpsServer(upstream, dexUpstream *url.URL, tlsSecretName string, secret
 		}
 
 		hostString, success := c.GetPostForm("hostname")
-		if success != true {
+		if !success {
```
Comment (emosbaugh) [inline kurl_proxy/cmd/main.go:282]: go lint error

File: kurl_proxy/cmd/main.go [hunk 4]
```diff
@@ -347,7 +347,7 @@ func getHttpsServer(upstream, dexUpstream *url.URL, tlsSecretName string, secret
...
```
Comment (emosbaugh) [inline kurl_proxy/cmd/main.go:350]: go lint error

**Review by salahalsaleh — APPROVED** [review]

**Pull request merged**